bc-backend/src/main/java/org/jboss/da/bc/backend/api/RepoClonator2000.java

```diff
@@ -0,0 +1,10 @@
+package org.jboss.da.bc.backend.api;
+
+/**
+ *
+ * @author Honza Brázdil <jbrazdil@redhat.com>
+ */
+public interface RepoClonator2000 {
```

Comment:
Good joke, but rebase it after https://github.com/project-ncl/dependency-analysis/pull/145 will be merged